content/blog/2020/06/2020-06-26-ui-ux-hackfest-ui-track.adoc

```diff
@@ -0,0 +1,138 @@
+* We contributed to the Jenkins UI accessibility, including UX testing and fixing the reported issues.
+  jira:JENKINS-62437[Configuration UI: Tables to divs migration] testing was the dominant story there.
+* We worked on a New Script Security approvals management UI 
+
+See the blog post below to know more about these and other user interface improvements.
+
+=== Read-only Jenkins Configuration
+
+link:http://www.youtube.com/watch?v=Wy8PdkS0kjQ[demo],
+link:https://github.com/jenkinsci/ui-ux-hackfest-2020/tree/master/presentations/05-system-read[presentation materials]
+
+A read-only view of Jenkins configurations, jobs and agents is important to plugin:configuration-as-code[Jenkins Configuration-as-Code] users.
+It would allow them to access configuration and diagnostics information about their Jenkins instances while having no opportunity to occasionally change it.
+This story is a part of the link:/project/roadmap[Jenkins roadmap],
+and it was featured as an area for contribution during the hackfest.
+
+In the beginning of the hackfest, on May 25th, we have released a preview for Read-only Jenkins Configuration.
+Read the announcement from Tim Jacomb link:/blog/2020/05/25/read-only-jenkins-announcement/[in this blogpost].
+During the hackfest we kept testing the change and fixing compatibility in the Jenkins plugins.
+
+We would appreciate feedback and testing from the Jenkins users! See the blogpost for the guidelines.
+
+image:/images/post-images/2020/05-read-only-jenkins-announcement/build-discarder-read.png[Read-only build discarder configuration]
+
+=== Dark Theme
+
+link:https://youtu.be/0ZLAOCs3XwU?t=958[demo]
+
+Dark user interface themes are very popular among developers: in IDE, communication tools, etc.
+And there is an interest to have one for Jenkins.
+There were a few of implementations before the hackfest, most notably link:https://github.com/camalot/jenkins-dark-stylish[camalot/jenkins-dark-stylish] and a dark version of the link:https://github.com/TobiX/jenkins-neo2-theme[Neo2 Theme].
+These themes were difficult to maintain, and finally they were either removed or abandoned. 
+What if Jenkins had an official theme?
+
+During the hackfest a group of contributors focused on creating a new link:https://github.com/jenkinsci/dark-theme-plugin[Dark Theme] for Jenkins.
+This effort included:
+
+* Patches to the Jenkins core which simplified development and maintenance of UI themes.
+  Support for CSS variables was added, as well as PostCSS processing which helps to simplify browser compatibility.
+* link:https://github.com/jenkinsci/dark-theme-plugin[Dark Theme] itself.
+* UI Testing and compatibility fixes in the core and multiple Jenkins plugins.
+* link:https://github.com/jenkinsci/dark-theme-plugin/tree/master/demo[Dark theme demo] with support for the development mode.
+
+You can try out this theme starting from Jenkins 2.239.
+It is also available as a plugin:dark-theme[plugin] from the Jenkins Update Center.
```

Comment:
I thought that it required that plugin in order to enable it.  If it is available separately without installing the plugin then you should ignore this suggestion.
```suggestion
It is available as a plugin:dark-theme[plugin] from the Jenkins Update Center.
```